index.html

```diff
@@ -1,86 +1,25 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-    <title>Ong Dev Blog</title>
-    <link
-      rel="stylesheet"
-      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
-    />
-    <script src="https://kit.fontawesome.com/e40be82818.js" crossorigin="anonymous"></script>
-    <link rel="stylesheet" href="resources/css/index.css" />
-    <link rel="stylesheet" href="resources/css/responsive.css" />
-    <link rel="stylesheet" href="resources/css/animation.css" />
-  </head>
-
-  <body>
-    <div id="cover"></div>
-    <div id="header">
-      <div id="brand-name">Ông Dev</div>
-      <div id="header-icon">
-        <i class="fa fa-bars" onclick="headerTogger()"></i>
-      </div>
-      <div id="header-container">
-        <div id="header-items">
-          <div class="item"><a href="#">Trang chủ</a></div>
-          <div class="item"><a href="#">Làm Dev</a></div>
-          <div class="item"><a href="#">Games</a></div>
-          <div class="item"><a href="#">Giới thiệu</a></div>
-        </div>
-      </div>
-      <div id="search">
-        <input id="search-input" type="text" placeholder="Search" />
-      </div>
-    </div>
-    <div class="posts-container">
-      <div id="feature-posts">
-        <h1>Featured Posts</h1>
-        Should we feature some popular posts here, make it a slider?
-      </div>
-      <div id="posts">
-        <h1>Posts</h1>
-      </div>
-      <div id="about">
-        <h1>About</h1>
-      </div>
-    </div>
-    <div id="footer">
-      <div class="posts-container footer">
-        <div class="social-icon">
-            <h3 class="title">Social</h3>
-            <ul class="list-unstyled icon">
-                <li><a href=""><i class="fa fa-facebook"></i></a></li>
-                <li><a href=""><i class="fa fa-twitter"></i></a></li>
-                <li><a href=""><i class="fa fa-instagram"></i></a></li>
-                <li><a href=""><i class="fa fa-youtube-play"></i></a></li>
-            </ul>
-        </div>
-        <div class="categories">
-            <h3 class="title">Categories</h3>
-            <div class="contain-box">
-                <a href="#" class="box">Custom</a>
-                <a href="#" class="box">Me</a>
-                <a href="#" class="box">Contact</a>
-                <a href="#" class="box">Custom</a>
-                <a href="#" class="box">Me</a>
-                <a href="#" class="box">Contact</a>
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Ông Dev Blog</title>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+    <main>
+        <section class="postSection">
```

Comment:
Name convention violated. Its name should be **post-section**.
Plase fix similar errors as well.

Reply:
Ok ok. I will check and fix after.